front/src/applications/operationalStudies/components/MacroEditor/osrdToNge.ts

```diff
@@ -139,18 +200,32 @@ const applyLayout = (state: MacroEditorState) => {
   const scaleY = 500;
   const padding = 0.1;
 
+  // Default positions: based on long/lat.
+  const nodes: [NodeIndexed, number, number][] = [];
```

Comment:
Maybe we could introduce `type PositionedNodeIndexed = NodeIndexed & { x: number; y: number }` instead of using a tuple?

Reply:
Ah, yeah, if we need to easily extract it without the `x`/`y` that makes sense to me!